src/screens/DataApi/DataCoverageTable.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import CoverageTable, {
+  COVERAGE_SUMMARY,
+} from 'screens/DataApi/DataCoverageTable';
+
+export default {
```

Comment:
nit: I would probably move the `DataCoverageTable` component to the `components` folder, but here is okay too